style.css

```diff
@@ -145,14 +193,34 @@ li {
   border-radius: 24px;
 }
 
-.portfolio-section {
+section.portfolio-section {
   padding: 16px;
   display: grid;
   justify-content: center;
   align-items: center;
   margin: 0 auto;
 }
 
+@media screen and (min-width: 992px) {
```

Comment:
- Isn't needed to have the same width on multiple times.
Kindly fix this and resume all in one media querie ``(min-width: 992px)``